docs/digger/admin/installing-jenkins-plugins.asciidoc

```diff
@@ -0,0 +1,21 @@
+== Installing Jenkins plugins
+Digger will by default install a number of Jenkins plugins required for its proper operation. If more Jenkins plugins are required, they can be specified in Ansible playbook(???) or using the Jenkins user interface.
```

Comment:
So, sorry I'm kinda lost.
- How is the jenkins_plugins defined/modified? Is there a command to modify the vars or just vi?
re: is there a possibility that user will unknowingly delete the default values?
"- so I think it is also fine to point them to the vars folder in this case too." here?
[defaults file](https://github.com/aerogear/digger-installer/blob/14aab018269a5e0613ceb07f26ba8fa4a46b8513/configure-buildfarm/defaults/main.yml#L13)